spec/lib/vmdb/settings_spec.rb

```diff
@@ -106,6 +106,18 @@
       )
     end
 
+    it "with a change to an Array" do
+      array_hash = {:log => {:collection => {:current => {:pattern => ["*.log"]}}}}
+      described_class.save!(miq_server, array_hash)
```

Comment:
This actually doesn't depend on the current value of the setting. This is testing that the correct entry is created in the `settings_changes` table.